lib/glific/providers/gupshup/message.ex

```diff
@@ -120,6 +120,8 @@ defmodule Glific.Providers.Gupshup.Message do
   @doc false
   @impl Glific.Providers.MessageBehaviour
   @spec receive_text(payload :: map()) :: map()
+  def receive_text(%{"sender" => %{"phone" => ""}}), do: {:error, "Phone number is blank"}
```

Comment:
```suggestion
  Let's write a test case for this.
```

Reply:
@pankaj-ag I don't see any test cases for this file